colossalai/fx/codegen/activation_checkpoint_codegen.py

```diff
@@ -1,14 +1,37 @@
-import colossalai
+from typing import Any, Callable, Dict, Iterable, List, Tuple
```

Comment:
This file I just modify the annotation of checkpoints.
`node.activation_checkpoint` into `node.meta['activation_checkpoint']`